src/application/view_rankings.c

```diff
@@ -0,0 +1,1153 @@
+    for (int i = 0; i < VIEW_RANKINGS_MISC_STRING_MAX; ++i) {
+        appData->miscStrings[i] = NewString_ReadMsgData(appData->msgData, msg_0421_00044 + i);
+    }
+}
+
+static void ViewRankings_DeleteStrings(ViewRankingsAppData *appData) {
+    for (int i = 0; i < VIEW_RANKINGS_MISC_STRING_MAX; ++i) {
+        String_Delete(appData->miscStrings[i]);
+    }
+    String_Delete(appData->playerNameString);
+    String_Delete(appData->rankingString);
+    String_Delete(appData->formatedStrBuf);
+    MessageFormat_Delete(appData->msgFormat);
+    DestroyMsgData(appData->msgData);
+}
+
+static void ViewRankings_PrintRecords(ViewRankingsAppData *appData) {
+    int i;
+    int ranking;
+    int y;
+    const RecordPageParam *pageParam;
+    ViewRankingsAppPage *pRecord;
+    int prev;
+    int cur;
+
+    pRecord   = &appData->records[appData->recordIdx];
+    pageParam = &sRecordPageParams[appData->page][appData->recordIdx];
+
+    FillWindowPixelBuffer(&appData->windows[VIEW_RANKINGS_APP_WINDOW_RECORDS], 0);
+    FillWindowPixelBuffer(&appData->windows[VIEW_RANKINGS_APP_WINDOW_RANKING], 0);
+    ranking             = 1;
+    y                   = 0;
+    prev                = 0;
+    i                   = 0;
+    appData->numRecords = 0;
+    for (; i < pRecord->count; ++i) {
+        if (pRecord->entries[i].active != 0) {
+            cur = min(pRecord->entries[i].pageEntry->stat, pageParam->max);
+            if (cur < prev) {
+                ++ranking;
+            }
+            BufferIntegerAsString(appData->msgFormat, 0, ranking, 1, PRINTING_MODE_RIGHT_ALIGN, TRUE);
+            StringExpandPlaceholders(appData->msgFormat, appData->formatedStrBuf, appData->rankingString);
+            AddTextPrinterParameterizedWithColor(&appData->windows[VIEW_RANKINGS_APP_WINDOW_RANKING], 0, appData->formatedStrBuf, 0, 16 * y, TEXT_SPEED_NOTRANSFER, MAKE_TEXT_COLOR(1, 2, 0), NULL);
+
+            BufferString(appData->msgFormat, 1, pRecord->entries[i].pageEntry->playerName, 2, 1, 2);
+            StringExpandPlaceholders(appData->msgFormat, appData->formatedStrBuf, appData->playerNameString);
+            AddTextPrinterParameterizedWithColor(&appData->windows[VIEW_RANKINGS_APP_WINDOW_RECORDS], 0, appData->formatedStrBuf, 0, 16 * y, TEXT_SPEED_NOTRANSFER, MAKE_TEXT_COLOR(1, 2, 0), NULL);
+
+            BufferIntegerAsString(appData->msgFormat, 0, cur, pageParam->numDigits, PRINTING_MODE_RIGHT_ALIGN, TRUE);
+            StringExpandPlaceholders(appData->msgFormat, appData->formatedStrBuf, appData->miscStrings[pageParam->msgId - msg_0421_00044]);
+            AddTextPrinterParameterizedWithColor(&appData->windows[VIEW_RANKINGS_APP_WINDOW_RECORDS], 0, appData->formatedStrBuf, 160 - FontID_String_GetWidth(0, appData->formatedStrBuf, 0), 16 * y, TEXT_SPEED_NOTRANSFER, MAKE_TEXT_COLOR(1, 2, 0), NULL);
+
+            appData->recordEntryIdxs[y] = i;
+            prev                        = cur;
```

Comment:
😟

Reply:
consequence of .clang-format AlignAssignments